provisioning/app-proxy/kubernetes/templates/proxy/deployment.yaml

```diff
@@ -0,0 +1,125 @@
+                  "histogram_buckets_as_distributions": true,
+                  "tag_by_endpoint": false
+                }
+              ]
+            }
+          }
+    spec:
+      containers:
+        - name: app-proxy
+          image: $APP_PROXY_REPOSITORY:$APP_PROXY_IMAGE_TAG
+          resources:
+            requests:
+              cpu: "200m"
+              memory: "768Mi"
+            limits:
+              memory: "768Mi"
+          ports:
+            - containerPort: 8000 # Proxy
+            - containerPort: 9000 # metrics
+          env:
+            # 80% of the resources.limits.memory
+            - name: GOMEMLIMIT
+              value: "550MiB"
+            - name: APP_PROXY_SANDBOXES_API_HOST
+              valueFrom:
+                configMapKeyRef:
+                  name: app-proxy
+                  key: sandboxesApiHost
+            - name: APP_PROXY_PUBLIC_ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: app-proxy
+                  key: appProxyApiHost
+            - name: APP_PROXY_COOKIE_SECRET_SALT
+              valueFrom:
+                secretKeyRef:
+                  name: app-proxy
+                  key: cookieSecretSalt
+            - name: DD_AGENT_HOST
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
+            - name: DD_TRACE_ANALYTICS_ENABLED
+              value: "true"
+            - name: DD_PROFILING_ENABLED
+              value: "true"
+            - name: DD_ENV
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/env']
+            - name: DD_SERVICE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/service']
+            - name: DD_VERSION
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['tags.datadoghq.com/version']
+          startupProbe:
+            httpGet:
+              path: /health-check
+              port: 8000
+              httpHeaders:
+                - name: Host
+                  value: KubernetesStartupProbe
+            initialDelaySeconds: 10
+            periodSeconds: 1
+            failureThreshold: 30
+          readinessProbe:
+            httpGet:
+              path: /health-check
+              port: 8000
+              httpHeaders:
+                - name: Host
+                  value: KubernetesReadinessProbe
+            periodSeconds: 10
+            failureThreshold: 1
+          livenessProbe:
+            httpGet:
+              path: /health-check
+              port: 8000
+              httpHeaders:
+                - name: Host
+                  value: KubernetesLivenessProbe
+            periodSeconds: 10
+            failureThreshold: 3
+      tolerations:
+        - key: app
+          operator: Exists
+          effect: NoSchedule
```

Comment:
tie `tolerations` tu tiez uz nemusia byt

Reply:
Tie tolerations teda mozeme tiez vyhodit z celeho repa.

Reply:
ano, ale radsej by som tu upravu/vyhodenie nespajal s tymto PR a poriesil to bokom v novom PR.

Reply:
nie je to ok :) je to genericka sablona, odstranit sa moze toto:
https://github.com/keboola/keboola-as-code/blob/eebcdc5872b0d54515e2ef815832c8f6b23d45fb/provisioning/common/etcd/values.yaml#L12-L15